tools/udisksctl.c

```diff
@@ -2601,20 +2601,15 @@ handle_command_dump (gint        *argc,
 static void
 monitor_print_timestamp (void)
 {
-  GTimeVal now;
-  time_t now_time;
-  struct tm *now_tm;
-  gchar time_buf[128];
-
-  g_get_current_time (&now);
-  now_time = (time_t) now.tv_sec;
-  now_tm = localtime (&now_time);
-  strftime (time_buf, sizeof time_buf, "%H:%M:%S", now_tm);
+  GDateTime *now = g_date_time_new_now_local ();
```

Comment:
Need to call `g_date_time_unref()` once done with it?

Reply:
Oops, fixed.